Watchman.Integrations/Logging/SerilogInitializer.cs

```diff
@@ -24,9 +20,9 @@ public static ILogger Initialize(IMongoDatabase mongoDatabase)
                 .Enrich.WithThreadId()
                 .Enrich.WithMachineName()
                 .Enrich.WithEnvironmentUserName()
-                .WriteTo.RollingFile(new JsonFormatter(closingDelimiter: ","), "logs/all/log-{Date}.json", shared: true, flushToDiskInterval: TimeSpan.FromSeconds(15), restrictedToMinimumLevel: LogEventLevel.Verbose)
-                .WriteTo.RollingFile(new JsonFormatter(closingDelimiter: ","), "logs/warningplus/log-{Date}.json", shared: true, flushToDiskInterval: TimeSpan.FromSeconds(15), restrictedToMinimumLevel: LogEventLevel.Warning)
-                .WriteTo.RollingFile(new JsonFormatter(closingDelimiter: ","), "logs/errorplus/log-{Date}.json", shared: true, flushToDiskInterval: TimeSpan.FromSeconds(15), restrictedToMinimumLevel: LogEventLevel.Error)
+                .WriteTo.File(new JsonFormatter(closingDelimiter: ","), "logs/all/log-{Date}.json", shared: true, flushToDiskInterval: TimeSpan.FromSeconds(15), restrictedToMinimumLevel: LogEventLevel.Verbose)
```

Comment:
czemu nie rolling file?

Reply:
bo to pochodzi z przestarzałej biblioteki, więc zrobiłem update nuget packages
ewentualnie mogę dodać jakiś RollingInterval ale nie wiem czy to konieczne

Reply:
fajnie byłoby gdyby był osobny plik dla każdego dnia
prościej wtedy coś znaleźć

Reply:
a to że jest `"logs/all/log-{Date}.json"` to już tego nie robi?
jeśli nie to mogę dać RollingInterval.Day